modules/javafx.controls/src/main/java/javafx/scene/control/TableView.java

```diff
@@ -452,7 +448,7 @@
         ConstrainedColumnResize.forTable(ConstrainedColumnResize.ResizeMode.AUTO_RESIZE_SUBSEQUENT_COLUMNS);
 
     /**
-     * A policy that tries to adjust columns, starting with the next one, in order to fit the table width.
+     * A resize policy that adjusts columns, starting with the next one, in order to fit the table width.
      * <p>
```

Comment:
Minor: you can remote this `<p>`, too.

Reply:
I liked it with <p> :-)

Reply:
It should at least be consistent with the rest. of them.

The formatting (with or without the `<p>`) is something we could ask others to weigh in on, too. I don't care too much, one way or the other.

Reply:
What I meant is that `TableView::CONSTRAINED_RESIZE_POLICY_FLEX_NEXT_COLUMN` and `TreeTableView::CONSTRAINED_RESIZE_POLICY_FLEX_NEXT_COLUMN` are now inconsistent, since the former has an extra `<p>` tag and the latter does not. They are otherwise identical.

As I said, I don't really care all that much. If you want to add them back in, that's fine (don't worry about updating the CSR for now).

Reply:
oh, sorry, I misunderstood.  fixed it now, thanks!